pkg/models/created_by_relation.go

```diff
@@ -0,0 +1,28 @@
+package models
+
+import (
+	"context"
+
+	"time"
+
+	"github.com/google/uuid"
+
+	"github.com/cmsgov/easi-app/pkg/appcontext"
+	"github.com/cmsgov/easi-app/pkg/authentication"
+)
+
+// CreatedByRelation is a struct meant to be embedded to show that the object has created_by and created_dts fields, as well as
+// shared logic to get the user account represented by the created_by uuid
+type createdByRelation struct {
+	CreatedBy  uuid.UUID `json:"createdBy" db:"created_by"`
+	CreatedDts time.Time `json:"createdDts" db:"created_dts"`
+}
+
+// CreatedByUserAccount returns the user account of the user who created the struct from the DB using the UserAccount service
+func (cbr *createdByRelation) CreatedByUserAccount(ctx context.Context) *authentication.UserAccount {
+
+	service := appcontext.UserAccountService(ctx)
+	account, _ := service(ctx, cbr.CreatedBy)
```

Comment:
```suggestion
	getUserAccount := appcontext.UserAccountService(ctx)
	account, _ := getUserAccount(ctx, cbr.CreatedBy)
```

I think naming this way makes it more clear what's happening!

Reply:
thanks Clay! Updated here and in modifiedByUserAccountRelation